bindata/v4.1.0/kube-apiserver/certregenerationcontroller-config-cm.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-apiserver-cert-regeneration-controller-config
+  namespace: openshift-kube-apiserver
+data:
+  kubeconfig: |
+    authentication:
+      disabled: true
+    authorization:
+      disabled: true
```

Comment:
Allowing unrestricted access to debug and metrics hasn't been vetted as viable and risks escalation bugs.  Can you shut off the entire server in the binary instead until the option can be plumbed?

Reply:
ok, disabled it